deploy/bundle/manifests/ocs-operator.clusterserviceversion.yaml

```diff
@@ -2724,6 +2726,8 @@ spec:
     name: ceph-container
   - image: quay.io/csiaddons/volumereplication-operator:v0.1.0
     name: volume-replication-operator
+  - image: quay.io/csiaddons/k8s-sidecar:v0.2.1
+    name: csiaddons-sidecar
```

Comment:
`csiaddons-k8s-sidecar` is more apt here

Reply:
k8s is not something we have used in rook. not sure it's important to use k8s. @nixpanic any thoughts?

Reply:
we don't use k8s in names am fine with one. leaving as it is